stat/distuv/poisson.go

```diff
@@ -129,7 +129,10 @@ func (p Poisson) StdDev() float64 {
 
 // Survival returns the survival function (complementary CDF) at x.
 func (p Poisson) Survival(x float64) float64 {
-	return 1 - p.CDF(x)
```

Comment:
I think the original is better. It is obviously correct and consistent with the documentation.

Reply:
But less accurate for x close to xm.